src/features/game/lib/processEvent.ts

```diff
@@ -123,6 +123,7 @@ export const MAX_ITEMS: Inventory = {
   Stone: new Decimal("1600"),
   Wood: new Decimal("8000"),
   "Wild Mushroom": new Decimal("100"),
+  "Magic Mushroom": new Decimal("100"),
```

Comment:
We usually try to put the FE Hoard limit to be lower than the BE, could you adjust this to 75?

Reply:
Is this situation the same for Wild Mushrooms? how much is their hoard limit on the backend?

Reply:
> @eliasSFL How much is the wild mushrooms hoard limit on the backend?

@Droidze 100